src/lib/consts.js

```diff
@@ -51,20 +53,17 @@ exports.ACTS_TEMPLATE_LIST = Object.keys(exports.ACTS_TEMPLATES);
 
 exports.DEFAULT_ACT_TEMPLATE = 'basic';
 
+exports.DEFAULT_LOCAL_STORES_ID = 'default';
+
 exports.LOCAL_ENV_VARS = {
```

Comment:
Is this needed for anything?

Reply:
I used it for creating path for input and output files in tests and in creating acts folders structure